fixed input box names so creds would be submitted properly

guitarmanj opened the pull request:
@zeroSteiner Creds weren't being submitted properly. Fixed by just adding names to the input boxes.

zeroSteiner commented:
I think I fixed one of these. Is this branch synced with master?

On Sat, Oct 17, 2015, 22:02 guitarmanj <notifications@github.com> wrote:

> @zeroSteiner <https://github.com/zeroSteiner> Creds weren't being
> submitted properly. Fixed by just adding names to the input boxes.
> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
>   https://github.com/securestate/king-phisher-templates/pull/3
> Commit Summary
>
>    - fixed input box names so creds would be submitted properly
>
> File Changes
>
>    - *M* Creds_Intranet/landing_page/forgotpassword.html
>    <https://github.com/securestate/king-phisher-templates/pull/3/files#diff-0>
>    (4)
>    - *M* Creds_Intranet/landing_page/login.html
>    <https://github.com/securestate/king-phisher-templates/pull/3/files#diff-1>
>    (6)
>
> Patch Links:
>
>    - https://github.com/securestate/king-phisher-templates/pull/3.patch
>    - https://github.com/securestate/king-phisher-templates/pull/3.diff
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/securestate/king-phisher-templates/pull/3>.
>

guitarmanj commented:
yessir

From: Spencer McIntyre [mailto:notifications@github.com]
Sent: Saturday, October 17, 2015 10:10 PM
To: securestate/king-phisher-templates <king-phisher-templates@noreply.github.com>
Cc: Jeremy J. Schoeneman <jschoeneman@securestate.com>
Subject: Re: [king-phisher-templates] fixed input box names so creds would be submitted properly (#3)

I think I fixed one of these. Is this branch synced with master?

On Sat, Oct 17, 2015, 22:02 guitarmanj <notifications@github.com<mailto:notifications@github.com>> wrote:

> @zeroSteiner <https://github.com/zeroSteiner> Creds weren't being
> submitted properly. Fixed by just adding names to the input boxes.
> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
> https://github.com/securestate/king-phisher-templates/pull/3
> Commit Summary
>
> - fixed input box names so creds would be submitted properly
>
> File Changes
>
> - *M* Creds_Intranet/landing_page/forgotpassword.html
> <https://github.com/securestate/king-phisher-templates/pull/3/files#diff-0>
> (4)
> - *M* Creds_Intranet/landing_page/login.html
> <https://github.com/securestate/king-phisher-templates/pull/3/files#diff-1>
> (6)
>
> Patch Links:
>
> - https://github.com/securestate/king-phisher-templates/pull/3.patch
> - https://github.com/securestate/king-phisher-templates/pull/3.diff
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/securestate/king-phisher-templates/pull/3>.
>


—
Reply to this email directly or view it on GitHub<https://github.com/securestate/king-phisher-templates/pull/3#issuecomment-148968349>.

guitarmanj commented:
Yeah when I just updated from the master, it still showed there were no “names” in those fields. Just wanted to make sure it was all good since it’s a public repo :P

From: Spencer McIntyre [mailto:notifications@github.com]
Sent: Saturday, October 17, 2015 10:10 PM
To: securestate/king-phisher-templates <king-phisher-templates@noreply.github.com>
Cc: Jeremy J. Schoeneman <jschoeneman@securestate.com>
Subject: Re: [king-phisher-templates] fixed input box names so creds would be submitted properly (#3)

I think I fixed one of these. Is this branch synced with master?

On Sat, Oct 17, 2015, 22:02 guitarmanj <notifications@github.com<mailto:notifications@github.com>> wrote:

> @zeroSteiner <https://github.com/zeroSteiner> Creds weren't being
> submitted properly. Fixed by just adding names to the input boxes.
> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
> https://github.com/securestate/king-phisher-templates/pull/3
> Commit Summary
>
> - fixed input box names so creds would be submitted properly
>
> File Changes
>
> - *M* Creds_Intranet/landing_page/forgotpassword.html
> <https://github.com/securestate/king-phisher-templates/pull/3/files#diff-0>
> (4)
> - *M* Creds_Intranet/landing_page/login.html
> <https://github.com/securestate/king-phisher-templates/pull/3/files#diff-1>
> (6)
>
> Patch Links:
>
> - https://github.com/securestate/king-phisher-templates/pull/3.patch
> - https://github.com/securestate/king-phisher-templates/pull/3.diff
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/securestate/king-phisher-templates/pull/3>.
>


—
Reply to this email directly or view it on GitHub<https://github.com/securestate/king-phisher-templates/pull/3#issuecomment-148968349>.
